### DISCUSSION TITLE
Export advanced pdf downloading in pages

### DISCUSSION THREAD

**Discussion by saibalajip** [jgraph/drawio#4378] [General]
Hi there,

I'm trying to export the diagram with advanced export as pdf option. It is getting downloaded but it has diagram broken into pages which is unusual

I previously downloaded using advanced pdf option which worked

But currently the previous diagram when I tried to download it is also getting failed

Please help. I've disabled page view before exporting

It's redirects to this link - https://convert.diagrams.net/node/export

I'm assuming there is some version related miss out in the URL which is redirecting to

Also, I'm using draw.io online one

Regards,
Sai

**Comment by davidjgraph** [comment]
Please use the standard PDF export, the advanced export is going to be removed.

**Reply by saibalajip** [reply]
Great thanks, it worked
But the margins option is not available

**Comment by FriedbertRidecell** [comment]
@davidjgraph, same problem for me, but standard PDF export doesn't solve the issue either. On the other side the local version (currently still 24.2.5 does NOT have the issue.

Any idea on how to fix this for the online version 24.3.1?

Thanks a lot.

Kind regards,
Friedbert

**Comment by alderg** [comment]
Fixed in v24.4.0 with the _Crop_ option checked using the standard _Export PDF_ dialog:

<img width="413" alt="Screenshot 2024-05-11 at 08 34 00" src="https://github.com/jgraph/drawio/assets/921200/93955340-60e2-46a2-b479-7b04de2e1adc">

Use https://clear.diagrams.net/ to get the latest version.
